packages/plugins/plugin-space/src/SpacePlugin.tsx

```diff
@@ -637,6 +646,7 @@ export const SpacePlugin = ({
                   .map((space) =>
                     constructSpaceNode({
                       space,
+                      selectable: state.values.selectableCollections,
```

Comment:
`navigable` perhaps? I still feel we use the term ‘select’ for too many orthogonal interactions…